src/message_filters/__init__.py

```diff
@@ -172,7 +172,7 @@ def getElemBeforeTime(self, stamp):
             return None
         return older[-1]
 
-    def getLastestTime(self):
+    def getLatestTime(self):
```

Comment:
This is part of the public API, we should tick tock the method. Do you mind to keep the older one with a deprecation warning ?

Reply:
I prefer the decotator `@deprecated`